scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -337,10 +340,20 @@ class FossId internal constructor(
                             severity = Severity.HINT
                         )
                         val summary = ScanSummary(
-                            startTime, Instant.now(), "", sortedSetOf(), sortedSetOf(), listOf(issue)
+                            startTime,
+                            Instant.now(),
+                            "",
+                            sortedSetOf(),
+                            sortedSetOf(),
+                            listOf(issue)
                         )
 
-                        val scanResult = ScanResult(provenance, details, summary)
+                        val scanResult = ScanResult(
+                            provenance,
+                            details,
+                            summary,
+                            mapOf(SCAN_CODE_KEY to scanCode)
+                        )
```

Comment:
Nit: This would still fit into a single line, and probably still is readable enough that way.